src/CombineUnixArchives.cmake

```diff
@@ -104,6 +107,23 @@ function(COMBINE_UNIX_ARCHIVES
       # Use relative paths to work around too-long command failure when building
       # on Windows in AppVeyor
       file(RELATIVE_PATH new_obj_file_path ${scratch_dir} ${new_obj_file_path})
+      
+      ####### remove this section if we no longer support MINGW
+      get_filename_component(ext ${new_obj_file_path} EXT)

```

Comment:
is MINGW actually working? if not, and likely won't?, let's remove it right away